DontPanicLabs.Ifx.Telemetry.Logger.Azure.ApplicationInsights/DontPanicLabs.Ifx.Telemetry.Logger.Azure.ApplicationInsights.csproj

```diff
@@ -34,6 +34,7 @@
     <ItemGroup>
       <PackageReference Include="DontPanicLabs.Ifx.Configuration.Local" Version="1.0.0" />
       <PackageReference Include="DontPanicLabs.Ifx.Telemetry.Logger.Contracts" Version="1.0.0" />
+      <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.23.0" />
```

Comment:
I had to double check here, since `ServerTelemetryChannel` is in the `WindowsServer` namespace, that this isn't limited to working just in Windows. Docs are not very explicit but reference things like "In environments other than Windows..."; also, I ran the code exercising `ServerTelemetryChannel` on my mac (see PR description) to validate.